src/daemon/client.py

```diff
@@ -22,7 +22,9 @@ def format_request(self, command, data=None):
         return request
 
     async def start(self):
-        self.websocket = await websockets.connect(self._uri)
+        self.websocket = await websockets.connect(
+            self._uri, max_size=(1024 * 1024 * 30)
```

Comment:
Did you mean 10MB or 30MB?

Reply:
30 MB. Pyl is already seeing sizes of 3MB in some messages (he has thousands of plots).